### ISSUE TITLE
CS8627 for missing generic constraint should offer a fix besides 'Upgrade to C# 8.0 *beta*'

### ISSUE THREAD

**Issue by jnm2** [dotnet/roslyn#31619]
Visual Studio 2019 Preview 1

```cs
public static class Foo
{
    public static T? Bar<T>(T bar)
}
```

At the end of the line and on `T?`, two fixes are offered that prompt to upgrade the current project or all projects to C# 8.0 due to CS8370 which is also reported now. But when I pressed Ctrl+., what I really had in mind was not having to type `↵⇥where TResult : struct` myself. A fix to add missing generic constraints would be handy.

This is just while typing the signature.  I could also imagine refactoring an existing method and wanting a fix to add a class constraint, a new constraint, or a type constraint.

**Comment by CyrusNajmabadi** [comment]
Closing out due to lack of feedback.
